os/dutils.sh

```diff
@@ -57,6 +60,64 @@ function FIND_DBGBINFILE()
 	KERNEL_DBG_BIN="tinyara$EXEEXT"
 }
 
+function TRAP_MENU()
+{
+	echo ======================================================
+	echo "  \"Run TRAP tool for\""
+	echo ======================================================
+	echo "  \"1. Default binary\""
+	echo "  \"2. Custom binary\""
+	echo "  \"x. Exit\""
+	echo ======================================================
+
+	read SELECT_BIN
+	case ${SELECT_BIN,,} in
+		1|default)
+			if [ -z "$(ls -A $BINDIR)" ]; then
+				echo "No output file, Build the code and run TRAP again."
+				exit 1
+			fi
+			echo "Enter the name of the crash log file:"
+			read LOG
+			cp $LOG $TRAPDIR
+			TRAP_RUN log $LOG
+			;;
+		2|manual)
+			echo "Enter the following wrt os directory"
+			echo "For example: ../build/output/bin/ OR bin/"
+			echo ""
+			echo "Enter the binary folder name:"
```

Comment:
There are two "enter ...". Is it necessary two?

Reply:
So the first one is to mention that specify all paths wrt to os directory
Or should I just append all the three read lines with `wrt os directory` like below??
`Enter the configuration file path wrt os directory`
`Enter the Make.defs file path wrt os directory`
`Enter the crash log file path wrt os directory`